New weapon icons

AmonShokhinAhmed opened the pull request:
Changed the look of the weapon icons and adjusted how colors are applied to them a bit. 
(Also blocked input during the brief period before the game begins because i quickly needed it for the video recordings)

DavidPgl commented:
New icons looking good! I would only increase the difference between selected and not selected weapon. Make the current selected timeline weapon icon more vibrant or decrease the color/brightness for the not selected ones (or both).

The wall icon color is very close to red and having no walls anymore is thereby hard to notice. Maybe change it or do you guys think the red number next to it is enough? @AmonShokhinAhmed @kb173 @incredibleLeitman

AmonShokhinAhmed commented:
I changed some stuff according to the feedback of @DavidPgl:
- I used saturation to make selected timeline's icon pop more, i hope this helps?
![image](https://user-images.githubusercontent.com/32668336/150567062-0d178af0-d6b8-42d7-918d-a2c6b4ebf691.png)
- I also changed the accent colors to be the same for both player and enemies, as this allowed me to use yellow for the wall accent and I think this actually might be more readable for new players as they can compare their own accent color with the one of the enemy to find out which weapon the enemy is currently using.
![image](https://user-images.githubusercontent.com/32668336/150567315-68e88cbd-d618-445c-9ab4-b8ada3dba3a2.png)
Do you guys think this works? @kb173 @incredibleLeitman

DavidPgl commented:
Looks awesome 🙏
